### DISCUSSION TITLE
Some directories missing for no obvious reason

### DISCUSSION THREAD

**Discussion by apa512** [nvim-tree/nvim-tree.lua#1727] [Q&A]
### Description

I recently switched to nvim-tree from nerdtree and for some reason that I can't figure out some directories are simply missing from the file browser. One such example is node_modules in a JS project that I'm working on. 

![2022-11-05-034505_3840x2332_scrot](https://user-images.githubusercontent.com/319058/200090126-a4123d43-9456-43be-8643-894d41f97fae.png)

![2022-11-05-034403_3840x2332_scrot](https://user-images.githubusercontent.com/319058/200090091-295530ca-d2d8-46b7-9691-c7a519f30ef8.png)

Is this a bug or am I simply missing something about how nvim-tree is supposed to work?

### Neovim version

```text
NVIM v0.8.0
Build type: Release
LuaJIT 2.1.0-beta3
```

### Operating system and version

Linux 5.15.75, Nixos 22.05

### nvim-tree version

a0f3e99

### Minimal config

```lua
lua << EOF
vim.g.loaded_netrw = 1
vim.g.loaded_netrwPlugin = 1

require("nvim-tree").setup()
EOF
```

### Steps to reproduce

cd ~/Code/my_project
nvim
:NvimTreeOpen

### Expected behavior

I should see all directories that are not hidden.

### Actual behavior

One directory is missing and I can't see anything such as permissions that's setting it apart from other directories.

**Comment by alex-courtis** [comment]
Is `node_modules` directory in `.gitignore`?

Git ignored files are hidden by default, see `:help nvim-tree.git.ignore`.

They can be toggled via `I` `toggle_git_ignored`. See `:help nvim-tree-default-mappings`.

**Comment by apa512** [comment]
Yes, thanks. I completely missed that this was a feature.

**Comment by alex-courtis** [comment]
You can also always show a particular path regardless of any ignores:

`:help nvim-tree.filters.exclude`
